src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -258,9 +375,42 @@ internal void Update(double number)
 
                         lock (this.histogramBuckets.LockObject)
                         {
-                            this.primaryValue.CurrentAsLong++;
-                            this.secondaryValue.CurrentAsDouble += number;
-                            this.histogramBuckets.BucketCounts[i]++;
+                            this.runningValue.AsLong++;
+                            this.histogramBuckets.RunningSum += number;
+                            this.histogramBuckets.RunningBucketCounts[i]++;
+                        }
+
+                        break;
+                    }
+
+                case AggregationType.HistogramWithMinMax:
+                    {
+                        int i;
+                        for (i = 0; i < this.histogramBuckets.ExplicitBounds.Length; i++)
+                        {
+                            // Upper bound is inclusive
+                            if (number <= this.histogramBuckets.ExplicitBounds[i])
+                            {
+                                break;
+                            }
+                        }
+
+                        lock (this.histogramBuckets.LockObject)
+                        {
+                            this.runningValue.AsLong++;
+                            this.histogramBuckets.RunningSum += number;
+                            this.histogramBuckets.RunningBucketCounts[i]++;
+
+                            if (this.runningValue.AsLong == 1)
+                            {
+                                this.histogramBuckets.RunningMin = number;
+                                this.histogramBuckets.RunningMax = number;
+                            }
+                            else
+                            {
+                                this.histogramBuckets.RunningMin = Math.Min(this.histogramBuckets.RunningMin, number);
+                                this.histogramBuckets.RunningMax = Math.Max(this.histogramBuckets.RunningMax, number);
+                            }
```

Comment:
If we had a default value for `this.histogramBuckets.RunningMin` and `this.histogramBuckets.RunningMax` then the `Math.Min` and `Math.Max` respectively might not work as intended, unless the defaults were `-Inf` and `+Inf`

Related to what should happen when calling `GetHistogramMin()` and `GetHistogramMax()` without recording any records.